dp-filter-api.nomad

```diff
@@ -0,0 +1,100 @@
+job "dp-filter-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  // Make sure that this API is only ran on the publishing nodes
+  constraint {
+    attribute = "${node.class}"
```

Comment:
Does this need to only run on publishing? The group below says web which seems accurate